scripts/ble_transfer.py

```
@@ -406,6 +424,225 @@ async def put_book(args: argparse.Namespace) -> int:
+                        done.set()
+                        return
+                    sequence = struct.unpack("<I", raw[:4])[0]
+                    if sequence != expected_sequence:
+                        data_error = f"unexpected data sequence: got {sequence}, expected {expected_sequence}"
+                        done.set()
+                        return
+                    payload = raw[4:]
+                    handle.write(payload)
+                    received_bytes += len(payload)
+                    expected_sequence += 1
+                    asyncio.create_task(write_json(client, {"op": "get_ack", "sequence": sequence}))
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)